backend-go/samples/without_nats/main.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+	"github.com/wuespace/telestion/backend"
+	"log"
+)
+
+func main() {
+	service, err := telestion.StartService(telestion.WithoutNats())
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.Println("Telestion service is running!")
+
+	log.Printf("Data directory: %s", service.DataDir)
```

Comment:
Let's add an explanation for when this might be relevant
```suggestion
    // Run without establishing a NATS connection. Especially useful during development
	service, err := telestion.StartService(telestion.WithoutNats())
	if err != nil {
		log.Fatal(err)
	}
	log.Println("Telestion service is running!")

	log.Printf("Data directory: %s", service.DataDir)
```